src/data/repositories/OptionsD2Repository.ts

```diff
@@ -24,6 +24,10 @@ export class OptionsD2Repository implements OptionsRepository {
             });
     }
 
+    getAllDataSources(): FutureData<Option[]> {
```

Comment:
please merge with Ana's feature/add-data-source-create-event branch and change the PR to merge into feature/add-data-source-create-event branch instead , so this change will not show up as a merge conflict